<!doctype html>
<html lang="en">

	<head>

		<meta charset="utf-8">

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<link rel="stylesheet" href="./dist/reveal.css">
		<link rel="stylesheet" href="./dist/theme/eal.css" id="theme">
	</head>
	<script src="https://d3js.org/d3.v4.min.js"></script>
	<script src="https://d3js.org/d3-timer.v1.min.js"></script>
	<script src="https://d3js.org/d3-random.v1.min.js"></script>
	<script src="https://d3js.org/d3-quadtree.v1.min.js"></script>
	<script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
	<style type="text/css">
    code {
      white-space: pre;
    }

    .MathJax_CHTML {
			color:rgb(100, 100, 100);
      font-size: 100% !important;
    }
		.container{
		display: flex;
		}
		.col{
		flex: 1;
		}


.vertical-center {
  margin: 0;
  position: absolute;
  top: 50%;
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
}


  </style>




	<body>

		<div class="reveal">

			<div class="slides">
				<!-- ########################################################## -->
				<section>
					 </br>
					<h3>Coarse-graining evolutionary dynamics models</h3>
					 <!-- <img src="./figures/euler.png" height="400"> -->
					 	<img src="./figures/cover.svg" style="opacity:0.4;filter:alpha(opacity=40);">
					<h3>Ethan Levien (Dartmouth College)</h3>
					<p>Joint work with Farshid Jafarpour (Utrecht University) </br>and Ariel Amir (Harvard University) </p>

				</section>


<!-- ########################################################## -->
	<section>
		<h2> Evolution: a problem of scales </h2>
	</br>
		<div class = "r-stack">
		<img  src="./figures/scales.png">
		<img class = "fragment" src="./figures/scales2.png">
	</div>
		<p> One of the central challanges to making quantitative predictions about evolutionary dynamics is that in order to understand evolution we need to connect processes on divergent scales: Mutations alter microscopic processes in the cell, but their fate is determined by population level dynamics.  </p>
		</section>

<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
		<section>
<!-- ########################################################## -->
        <section>
				<h2> Part 1: The population scale </h2>
					<div class="r-stack">
				<img src="./figures/turbidostat.png" height="450" data-fragment-index="1">
				<img class="fragment current-visible" src="./figures/eVOLVER.png" height="400" data-fragment-index="3">
			</div>
				<!-- <p>The turbidostat is a type of Continuous culture in which the population size is kept approximately constant through feedback between an optical density detector and a pump </p> -->
				<p class = "fragment" data-fragment-index="1">Let us begin by introducing evolutionary dynamics models without reference to the single-cell scale.  </p>
				<p class = "fragment" data-fragment-index="2">To keep the discussion biologically grounded, we will imagine evolution playing out in a continous culture.  </p>
			</section>



<!-- ########################################################## -->
			<section>
			<h2> Evolution in the turbidostat </h2>
			<p>
		The basic dynamics in a continous culture can be understood from the ODE model for the number of cells of each genotype $n_i$:
		\[
			\frac{dn_i}{dt} = \underbrace{\rho_i(c)n_i}_{\text{growth}} - \underbrace{Dn_i}_{\text{dilution}}
			\]</p>
			<p class = "fragment" data-fragment-index="2">
				The growth rates depend on a limiting nutient:
				\[
			\frac{dc}{dt} = \underbrace{Dc_0}_{\text{media in}}- \underbrace{Dc}_{\text{dilution}}
- \underbrace{\gamma^{-1}\left[\sum_i\rho_i(c)n_i\right]c}_{\text{nurtient update}}
\] </p>

<div class="r-stack">
			<p class="fragment current-visible" data-fragment-index="3">In the <strong> turbidostat </strong> the dilution rate $D$ is adjusted to fix the total population size $N = \sum_i n_i$, which means
			\[
			\begin{equation}
			D = \rho_i(c)n_i \implies c  \to \frac{\gamma c_0}{\gamma+1}
			\end{equation}\]
		</p>

		<p class="fragment current-visible" data-fragment-index="4"> We obtain the so-called replicator equations for the genotype fractions $\phi_i = n_i/N$:
			\[
			\frac{d\phi_i}{dt} = \phi_i\left(\rho_i - D - \sum_j \phi_j(\rho_j-D)\right)
		\]
	</p>

	<p class="fragment current-visible" data-fragment-index="5"> We will focus on the case of two species, a "mutant" with growth rate $\rho_m$ and "resident" with growth rate $\rho_r<\rho_m$:
		\[
		\frac{d\phi}{dt} = (\rho_m - \rho_r)\phi(1-\phi)
	\]
</p>
	</div>

				</section>


<!-- ########################################################## -->
				<section>
				<h2> The Moran model </h2>
			</br>
				<p>These equations work well when $\phi$ is not too small, but new mutations emerge in single-cells, so $\phi(0) = 1/N$ $\implies$ Model must include fluctuations! </p>

				<h4 class = "fragment">Question: How do we add noise to the model?  </h4>

				<div class="r-stack">
				<div class = "fragment current-visible">
				<p > <strong> Modeling noise in the (continous time) Moran model </strong> <p>
					<p> <ul>
						<li> <strong> Sampling</strong>: We assume each cell has equal chance of being expelled in a dilution event. </li>
						 <li> <strong>Cell-to-cell variability</strong>: We assume cells divide at a constant rate $\rho_i$; that is, $\rho_idt$ is the chance for a cell of genotype $i$ to divide in the interval $[t,t+dt)$. </li>

			</ul></div>

	<img class = "fragment current-visible" src="./figures/cover.svg">
		</div>
				</section>


<!-- ########################################################## -->
				<section>
					<h2> The Moran model </h2>
					<p class = "fragment" data-fragment-index="1"> This means that over a small time interval $\Delta t$, the number of mutants gained and lost are both given by Poisson distributions:
					\[\begin{aligned}\text{# of mutants gained} &\sim {\rm Poission}(N\rho_m\phi(1-\phi) \Delta t)\\
					\text{# of mutants lost} &\sim {\rm Poission}(N\rho_r\phi(1-\phi) \Delta t)
					\end{aligned}\]</p>

					<div class="r-stack">
					<p class="fragment current-visible" data-fragment-index="3"> We can calculate the mean and variance of $\Delta \phi$:
						\[\begin{aligned}\color{rgb(20, 116, 111)}{\langle  \Delta \phi \rangle} &= (\rho_m - \rho_r)\phi(1-\phi)\Delta t\\
						\color{rgb(205, 88, 73)}{{\rm var} \left(\Delta \phi \right)} &= \frac{(\rho_m + \rho_r)}{N}\phi(1-\phi)\Delta t
					\end{aligned}\]
			</p>
				<p class="fragment" data-fragment-index="4">
					Corresponds to SDE
					\[
					\frac{d}{dt}\phi = \color{rgb(20, 116, 111)}{\underbrace{(\rho_m-\rho_r)\phi(1-\phi)}_{\text{selection}}} + \color{rgb(205, 88, 73)}	{\underbrace{\sqrt{(\rho_m + \rho_r)\frac{\phi(1-\phi)}{N}}\xi}_{\text{fluctuations/drift}}}
					\]
				</p></div>

				<p class="fragment" data-fragment-index="4">
					or the Fokker-Plank equation
					\[
					\frac{\partial}{\partial t}p(\phi,t) = \color{rgb(20, 116, 111)}{-\frac{\partial}{\partial \phi}[(\rho_m - \rho_r)\phi(1-\phi)p(\phi,t)]} + \color{rgb(205, 88, 73)}	{\frac{1}{2}\frac{\partial^2}{\partial \phi^2}\left[\frac{(\rho_m + \rho_r)\phi(1-\phi)}{N}p(\phi,t) \right]}
					\]
				</p>

				</section>

				<!-- ########################################################## -->
				<section>
				<h2> The Moran model </h2>
						<p> Some trajectories starting on $\phi(0) = 1/N$  </p>
						<div class="r-stack">
						<img class="fragment" src="./figures/moran_fix1.svg" data-fragment-index="1">
					</div>
						<p class="fragment" data-fragment-index="2" > The behavoir depends stronly on whether selection or drifts dominates. </p>
						<h4 class="fragment" data-fragment-index="4" > When does the transition occur? </h4>

			</section>


			<!-- ########################################################## -->
			<section>
			<h2> The Moran model </h2>

		</br>
			<p> We know it must be when $\phi$ is small, so we can linearize
				\[
		 \frac{d\phi}{dt} =\delta \rho\phi(1-\phi) + \sqrt{2\rho_r\frac{\phi(1-\phi)}{N}} \approx \delta \rho \phi + \sqrt{2\rho\phi/N}\xi
						\]
			 </p>




			<p class = "fragment">
				And calculate moments of linearized equation
				\[\frac{d}{dt}\left\langle\phi^k \right\rangle = \int \phi^k \frac{\partial}{\partial t}p(\phi,t)d\phi \to \text{integrate by parts}\]
				<!-- \[
				\begin{aligned}
				\langle \phi \rangle &=  \phi_0e^{\delta\rho t}\\
					\langle \phi^2 \rangle &= \left[\phi_0 e^{\delta \rho t} \left(e^{\delta \rho t} (N \delta \rho \phi (0)+1)-1\right) \right]/(N \delta \rho)\\
					\end{aligned}
				\] -->
				 </p>

		 <p class = "fragment">
			which lead to an equation for the coefficient of variation (CV):
			 \[
		    CV = \frac{{\rm std}(\phi)}{\langle \phi\rangle} =\frac{2\rho_r(1-e^{-st})}{Ns\phi_0} \to \frac{2}{Ns \phi_0},\quad s = \frac{\delta \rho}{\rho_r}
		  			\]
				</p>

				<p class = "fragment">
				The quantity $\color{rgb(205, 88, 73)}{1/Ns}$ is called the <strong> Drift barrier </strong>
			</p>
			</section>


			<!-- ########################################################## -->
			<!-- <section>
			<h2> Properties of the Moran model </h2>

					<div class="r-stack">
					<img class="fragment" src="./figures/moran_fix1.svg" data-fragment-index="1">
					<img class="fragment" src="./figures/moran_fix2.svg" data-fragment-index="3" >
				</div>
					<div class="r-stack">
					<div class="fragment current-visible" data-fragment-index="2" > <p> <strong> Fluctuations dominate: </strong> In this regime, $\langle \phi \rangle= 1/N$, but conditioned on the mutant lineage being alive at time $t$, we $\phi$ will be on the order of $t/N$.
					</p>
						<p> But if $\phi=0$ mutant is extinct forever, so really we expect $\phi = O(t/N\phi_0)$ or $\phi$ is extinct. Each unit time the mutant is equally likely to go exitinct or grow, so the chance of surviving until time $t$ is $1/t$. </p>
					</div>

					<div class="fragment current-visible" data-fragment-index="3" > <p> <strong> Selection dominates: </strong> This just logistic growth. </p>
						<p> <strong> Fxiation probability: </strong> The analysis tells us the chance to reach the drift barrier is $1/s$ </p>
					</div>
				</div>
		</section> -->



		<!-- ########################################################## -->
		<section>
		<h2> The Moran model </h2>
					<h4>So what is the chance the mutant lineage reaches the drift barrier before going extinct?</h4>


		<p class="fragment">
		Heuristic argument from Fisher 2007: Below the drift barrier, we can approximate the dynamics over a time $\Delta$ by
			\[
			\Delta \phi = \phi(t+\Delta)-\phi(t) = \sqrt{\frac{2\rho_r \Delta\phi(t)}{N}}Z,\quad Z \sim {\rm Normal}(0,1)
			\]
			This will approximation will give us an order-of-magnitude estimate of $\Delta \phi$ before some time $\Delta_1 = O(\phi(t)N/\rho_r)$.
		</p>

		<p class="fragment">
		 At $\Delta_1$, the mutant fraction will have changed by $\pm \phi(t)$. This means the population has roughly an equal chance of doubling or going extinct over a time period on the order of $\phi(t)N/\rho_r$. 	</p>




	</section>

	<!-- ########################################################## -->
	<section>
	<h2> The Moran model </h2>
				<h4>So what is the chance the mutant lineage reaches the drift barrier before going extinct?</h4>


	<img src = "./figures/fix_iter_drawing.png" height=400>





</section>


	<!-- ########################################################## -->
	<section>
	<h2>  The Moran model </h2>
				<h4>So what is the chance the mutant lineage reaches the drift barrier before going extinct?</h4>

<p >
	Repeat the approximation and after $k$ iterations a time $t_k$ has elapsed where
	\[
	t_k = t_{k-1} + \phi(t_{k-1})N/\rho_r = \frac{1}{\rho_r}\sum_j 2^{j} \sim \frac{2^k}{\rho_r} \implies k = \ln(\rho_rt)/\ln(2)
	\]
</p>
	<p class = "fragment">
Plugging $k = \ln(t\rho_r)/\ln(2)$ we are obtain order-of-magnitude estimates of the mutant size and chance of survival:
	\[
	\phi(t) = \left\{ \begin{array}{cl}
	 t\rho_r/N & \text{with prob. $\frac{1}{t\rho_r}$}\\
		0 & \text{with prob. $1-\frac{1}{t\rho_r}$}\end{array}\right.
	\]</p>
	<p class = "fragment">
	So the population will reach a fraction $1/Ns$ after a time $t_{\rm drift} = 1/s\rho_r$.</p>
	<p class = "fragment"> The chance the survive until this time is $p_{\rm fix}\sim 1/t_{\rm drift}\rho_r = s$.
</p>

</section>


<!-- ########################################################## -->
<section>
<h2> The Moran model </h2>
	<h4>How fast do mutations accumulate?</h4>
	<p class = "fragment">In general, very complicated problem because we need to consider competition between >2 mutants and the theory above breaks down. </p>

	<h4 class = "fragment"> When can we neglect competition between mutant lineages? </h4>

	<p class = "fragment"> Two important time-scales:</p>
		<ul>
			<li class = "fragment"><strong> Establishment time $\tau_{\rm est}$</strong>: The time for a new mutation to occur and become established (reach the drift barrier) in the population. Mutations happen at a rate $\mu$ per cell-division and a fraction $p_{\rm fix} \sim s$ of them will reach the dirft barrier, so $\tau_{\rm est}  = 1/Ns\rho_r\mu$.  </li>
		</br>
				<li class = "fragment"><strong> Sweep time $\tau_{\rm sw}$</strong>: The time for an established mutation to sweep through the population. Since established mutations grow approximately exponentially starting from the drift barrier, $\tau_{\rm sw}$ is on the order of $1/\delta \rho\ln (Ns)$  </li>
		</ul>



</section>

<!-- ########################################################## -->
<section>
<h2> The Moran model </h2>
	<h4>How fast do mutations accumulate?</h4>
	<p>In general, very complicated problem because we need to consider competition between >2 mutants and the theory above breaks down. </p>

	<h4> When can we neglect competition between mutant lineages? </h4>
		<img src = "./figures/succ_mutation_drawing.png" height=400>
	</section>


<!-- ########################################################## -->
<section>
<h2> The Moran model </h2>
	<h4>How fast do mutations accumulate?</h4>
	<p>In general, very complicated problem because we need to consider competition between >2 mutants and the theory above breaks down. </p>

	<h4> When can we neglect competition between mutant lineages? </h4>
	<p> Condition for "succesive mutations":
	\[\tau_{\rm sw} = 1/\delta \rho\ln (Ns) \ll \tau_{\rm est} = 1/N\mu \rho_r s \implies N\mu \ln Ns \ll 1 \]</p>

	<p class = "fragment">In this regime, fixations are Poisson process with rate $1/\tau_{\rm est} = N\rho_r\mu s$. </p>
		<p class = "fragment"> Simple case where all mutation increase fitness by $\delta \rho$:
	\[ \frac{d}{dt}\langle\rho \rangle = \frac{\delta \rho}{\tau_{\rm est}} = N\mu (\delta \rho)^2\implies \langle \rho(t) \rangle \sim tN\mu (\delta \rho)^2\] </p>
</section>


</section>

<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<section>
	<!-- ########################################################## -->
		<section>
			<h2> Part 2: From single-cell to population scale </h2>

			<img src="./figures/scales2.png">
		</br>
			<p class = "fragment">Underneath the Moran model was a pretty strong assumption about how noise in the growth and division of single-cell propogates to the population level.   </p>

				<p class = "fragment">We will revist this assumption in the context of the Moran model, but first we need to understand some basic aspects of so-called <strong>age-structured populations</strong>.  </p>


			</section>

					<!-- ########################################################## -->
					<section>
					<h2>Motivation from single-cell data</h2>

					<div class = "r-stack">
						<div class = "fragment current-visible">
									<iframe width = 900 height="220"
					src="https://www.youtube.com//embed/5PyoO4S3BU0">
					</iframe>
					<p>  Wang et al. Current Biology (2010) </p></div>
					<img class = "fragment" src="./figures/mm_drawing.png" height="300">
				</div>

					<p class = "fragment"> The basic picture of single-cell growth: cells accumulate mass exponentially at a (<strong>random</strong>) rate $\rho$ between cell divisions, then divide in two after a (<strong>random</strong>) time $\tau$. Complex regulatory mechansims shape the distribution of genertaions times
						<!-- \[s_{b,k} = \frac{1}{2}s_{b,k-1}e^{\rho\tau} \implies \ln s_{b,k} = \rho \tau -\ln(2) + \ln s_{b,k-1}\]
						Generation time must be coupled to initial cell-size and growth rate in order to mantain a stable distribution of cell-sizes -->
					</p>
					<h4 class = "fragment">
						Question: How does evolution shape the distribution the distribution of generation times?
					</h4>

					</section>

					<!-- ########################################################## -->
					<!-- <section>
						<h2>Model of single-cell growth</h2>
						<p> There is a large body of liturature seeking to understand the dynamics of single-cell growth and division. Key observations:
							<ul>
								<li> In fast growing bacteria (e.g. E coli), cell's accumulate biomass exponentially </li>
								<li> Cell-size is regulated (after a long time the population converges to a stable size distribution)</li>
							</ul>
					</section> -->


				<!-- ########################################################## -->
				<section>
				<h2> From single-cell to population scale </h2>
				<p class = "fragment"> Before we connect single-cell dynamics to evolution, let's first connect to population growth. </p>
				<div class = "fragment">	<p> Alfred Lotka (1934), Bellman and Harris (1948) and Powell (1956): Each cell's generation time is drawn <b>independently</b> from a distribution $f(\tau)$.</p>	</div>

				<div class = "fragment">
				<div class="viewof-sigma" class="horizontal-center"></div>
				<div class="tree" ></div>
				<script type="module">
				import {Runtime, Inspector} from "https://cdn.jsdelivr.net/npm/@observablehq/runtime@4/dist/runtime.js";
				import define from "https://api.observablehq.com/@elevien/branching-process/2.js?v=3";
				(new Runtime).module(define, name => {
					if (name === "viewof sigma") return Inspector.into(".viewof-sigma")();
					if (name === "tree") return Inspector.into(".tree")();
					return ["tree_data1"].includes(name) || null;
				});
				</script></div>


				<div class = "fragment">
				<p> In the long-term, ${\mathbb E}[N(t)] = Ae^{\rho t}$ for some $\rho$. </p>
				</div>
				<div class = "fragment">
				<h4> How does the rate at which the population grows depend on the distribution of generation times? </h4>
				</div>



				</section>




				<!-- ########################################################## -->
				<section>
				<h2> The Euler-Lotka Equation </h2>

				<div class="recursive_tree"></div>
				<script type="module">
				import {Runtime, Inspector} from "https://cdn.jsdelivr.net/npm/@observablehq/runtime@4/dist/runtime.js";
				import define from "https://api.observablehq.com/@elevien/branching-process/2.js?v=3";
				(new Runtime).module(define, name => {
				if (name === "recursive_tree") return Inspector.into(".recursive_tree")();
				});
				</script>

				<div class="fragment">
				<p>The root cell has two daughters with progenies that grow exponentially at the same rate $\rho$. </p></div>


				<div class="fragment"><p> Let $\tau$ be the first cell's generation time:
				<script type="math/tex; mode=display">
				{\mathbb E}\left[N_{\rm daughter}|\tau\right] = Ae^{\rho(t-\tau)} \implies
				{\mathbb E}\left[N_{\rm daughter}\right] = 	A \int_0^{\infty} e^{\rho(t-\tau)}f(\tau)d\tau
				</script>
			</p></div>


				<div class="fragment">
				<p> Writing the population size at time $t$ in terms of the size of both daughter progenies gives the so-called Euler-Lotka Formula
					<script type="math/tex; mode=display">
				Ae^{\rho t} = 	\color{rgb(20, 116, 111)}{{\mathbb E}\left[N_{\text{left daughter}}\right]} + \color{rgb(205, 88, 73)}	{{\mathbb E}\left[N_{\text{right daughter}}\right]}  = 2 A \int_0^{\infty} e^{\rho(t-\tau)}f(\tau)d\tau \implies \color{rgb(5, 5, 5)}{	\frac{1}{2} = \int_0^{\infty}f(\tau)e^{-\rho \tau}d\tau}
					</script>
				</p>
			</div>

				</section>


				<!-- ########################################################## -->
				<section>
					<h2> The Euler-Lotka Equation </h2>
						<div class = "r-stack">
					<p class = "fragment current-visible"> Example: assume generation time distribution is a Gamma distribution characterized by a mean, $\bar{\tau}$ and coefficient of variation, ${\rm CV}$:
					\[
				f(\tau) = \frac{1}{\Gamma(k)}\tau^{k-1}e^{-\tau/\theta}
						\]
						where $\theta = \langle\tau \rangle {\rm CV}^2$ and $k = 1/{\rm CV}^2$.
					</p>
						<img class = "fragment" src="./figures/gamma_dist.svg"></div>

<div class = "r-stack">
						<p class = "fragment current-visible">Plugging into the Euler-Lotka Formula reveals
						\[\rho = \frac{2^{-{\rm CV}^2}-1}{{\rm CV}^2\tau}\]</p>


					<div  class = "fragment">
					<p>
						Note that $\rho$ increases with ${\rm CV}^2$. Two important cases:
						<ul>
							<li> $CV \to 0$, $\rho \to \ln(2)/\langle\tau \rangle$ </li>
							<li> $CV =1$, $\rho = 1/\langle \tau \rangle$ </li>
						</ul><p></div>
						</div>
				</section>

				<!-- ########################################################## -->
				<section>
					<h2> Generalized Euler-Lotka Formula  </h2>

					<p>	The Euler-Lotka formula has an elagent generalization to the case where cell's generation times are correlated with their ancestors (Lebowitz et al., 1974):
						<script type="math/tex; mode=display">
							\frac{1}{2} = \int_0^{\infty} \color{rgb(20, 116, 111)}{f_{\rm tree}(\tau)}e^{-\rho \tau}d\tau = {\mathbb E}_{\rm tree}\left[e^{-\rho \tau}  \right].
						</script>
						where $\color{rgb(20, 116, 111)}{f_{\rm tree}(\tau)}$ is the distribution of generation times <b> taken over all cells </b> throughout the history of an exponentially growing population.
					</p>

									<div class="fragment">		<div class="tree_vs_lineage"></div>

						<script type="module">
						import {Runtime, Inspector} from "https://cdn.jsdelivr.net/npm/@observablehq/runtime@4/dist/runtime.js";
						import define from "https://api.observablehq.com/@elevien/lineage-representation.js?v=3";
						(new Runtime).module(define, name => {
						if (name === "tree_vs_lineage") return Inspector.into(".tree_vs_lineage")();
						});
						</script>
						<p> In general $\color{rgb(20, 116, 111)}{f_{\rm tree}(\tau)} \ne  \color{rgb(205, 88, 73)}{f(\tau)}$ </p></div>

							<p class= "fragment"> <strong>Interesting question</strong>: Can you infer $\Lambda$ from mother-machine lineage data? See Levien, GrandPre, Amir PRL 2020. </p>


						</section>


				<!-- ########################################################## -->
					<section>
						<h2> Exponential growth rate and age distribution </h2>
						<p> Another way to think about the exponetial growth rate: it is the
							average division rate over all cells in the population.</p>
							<p class = "fragment"> If $f_i(\tau)$ is the distribution of single-cell generation times conditioned on the growth rate and cell-size, the per unit time probability for the cell to divide is	\[
							\gamma_i(a) = \frac{f_i(a)}{1- \int_0^a\,da'\,  f_i(a')} \]
							Note special case: $f_i = \rho_ie^{\rho_i\tau}\implies \gamma_i = \rho_i$.
						</p>
						<p class = "fragment">Then we have (in a large population)
						\[\frac{d}{dt}N = \rho N = \langle \gamma_i(a) \rangle_h N\]
					where the average is taken over the distribution of ages $h(a)$ in the population.  </p>
					</section>

					<!-- ########################################################## -->
						<section>
							<h2> Exponential growth rate and age distribution </h2>
									<img src = "./figures/age_dist_drawing.png" height=400>
										<h4 class = "fragment"> What about a finite population? </h4>
						</section>



				<!-- ########################################################## -->
				<section>
				<h2> Finite populations </h2>

					<div class = "r-stack">
							<!-- <img class = "fragment current-visible" src = "./figures/age_dist_samp_drawing.png" height=300> -->
					<p class = "fragment">
					Sampling preserves age distribution, in the limit $N\to \infty$ the age distribution will have converged and we get
					\[\frac{d}{dt}\phi  = (\rho_m-\rho_r)\phi(1-\phi)\quad\quad{\rm with}\quad\quad  \frac{1}{2} = \int_0^{\infty}f_i(\tau)d\tau, \quad i \in \{m,r\}\]
			Connecting growth rates to single-cell statistics is actually more subtle; see Levien, Kondev, Amir 2019</p></div>
		</br>
			<h4 class = "fragment">Same question as before: how do we add noise to this? </h4>
			<div class = "r-stack">
			<p class = "fragment current-visible">We want to understand how the stochastic dynamics of birth and death at the single-cell level propogate to population level, but in this case cells do not divide at a constant rate. We need to know the age of every cell in the population to obtain the division rate! </br></br>
			We can try writing down a "master equation" which completely specifies the distribution of ages and genotypes. </p>



		<p class="fragment current-visible">	We then get a hyperbolic PDE for $Q$:\[
	 \frac{d}{dt} Q({\bf u},{\bf a},t) + \sum_{k=1}^N \frac{\partial}{\partial a_k} Q({\bf u},{\bf a},t)
	 = - Q({\bf u},{\bf a},t)\left(D+ \sum_{k=1}^N \gamma_{u_k}(a_k)\right) \]
	along with the boundary conditions
	 \[
	Q(S_{k,u_v}({\bf u}),S_{k,0}(S_{v,0}({\bf a})),t)= \frac{1}{N} \gamma_{u_v}(a_v)Q({\bf u},{\bf a},t)\]</p>

	</div>

			</section>




						<!-- ########################################################## -->
						<section>
						<h2> Bridging-scales </h2>
						 <p> We don't really care about $Q$ though! we are interested in the behavoir of the mutant fraction $\phi$. </p>
						<h4 class="fragment"> Question: can we coarse-grain to get an effective SDE?</h4>

						<p class = "fragment"> Just like before, we can try deriving an SDE:
						<!-- \[\begin{aligned}\text{# of mutants gained} &\sim {\rm Poission}\left(N\left[\sum_{j:u_j=m}\gamma_j(a_j,\rho_j,s_j) \right]\phi(1-\phi) \Delta t\right)\\
						\text{# of mutants lost} &\sim {\rm Poission}\left(N\left[\sum_{j:u_j=r}\gamma_j(a_j,\rho_j,s_j) \right]\phi(1-\phi) \Delta t\right)
						\end{aligned}\]-->
						\[\begin{aligned}\text{# of mutants gained} &\sim {\rm Poission}\left(N\langle\gamma_m(a_i) \rangle_h\phi(1-\phi) \Delta t\right)\\
						\text{# of mutants lost} &\sim {\rm Poission}\left(N\langle\gamma_r(a_j) \rangle_h\phi(1-\phi) \Delta t\right)
						\end{aligned}\]</p>

						<p class="fragment">
						We might guess that we can make the approximation $\langle\gamma_i(a_j,\rho_j,s_j) \rangle_h \approx \rho_i$. This obviously isn't going to work when $N\phi$ is very small, but perhaps if the drift barrier $1/s$ is not too small we can capture the dynamics with the SDE:
						\[
						\frac{d}{dt}\phi = (\rho_m-\rho_r)\phi(1-\phi) + \sqrt{(\rho_r + \rho_m)\frac{\phi(1-\phi)}{N}}\xi
					\]
					All the classical results about fixation and neutral drift then follow.
				</p>

				<p class ="fragment"> Much of the previous liturature on evolution in age-structured populations focuses on this approximation  (e.g. Shpak (2007))</p>



			</section>
		</section>


				<!-- ########################################################## -->
				<!-- ########################################################## -->
				<!-- ########################################################## -->
				<!-- ########################################################## -->
				<!-- ########################################################## -->
				<!-- ########################################################## -->
					<section>

					<!-- ########################################################## -->
					<section>
					<h2>Results: Neutral dynamics </h2>


					<p> Let's start by considering the neutral case ($\rho_m = \rho_r=0$). In the classical Moran model, we expect that
					\[
					\frac{d}{dt}\phi = \sqrt{\frac{2\rho}{N}\phi(1-\phi)} \implies {\rm var}(\phi) \approx \frac{2\rho}{N}\phi_0(1-\phi_0)t
				\]
					</p>

						<img class = "fragment" src="./figures/drif_ratio.png" height = 300>

				</br>
						<div class = "r-stack">
				<p class = "fragment current-visible"> Ratio is $O(1)$ $\implies$ effects of age-structure do not vanish in large populations.  What's going on? </p>
				<p class = "fragment current-visible"> <strong>Basic idea:</strong> age distribution gets smoothed out as $N$ grows, but overall strength of genetic drift decreases, so large $N$ means small effects matter. </p>
					<p class = "fragment">In limit ${\rm CV} \to0$ we get an extra factor of $3/8N\ln(2)$. </p>
				</div>
					</section>


					<section>
					<h2>Results: Neutral dynamics </h2>


					<p> Let's start by considering the neutral case ($\rho_m = \rho_r=0$). In the classical Moran model, we expect that
					\[
					\frac{d}{dt}\phi = \sqrt{\frac{2\rho}{N}\phi(1-\phi)} \implies {\rm var}(\phi) \approx \frac{2\rho}{N}\phi_0(1-\phi_0)t
				\]
					</p>

						<img src="./figures/oscil.png" height = 300>

				</br>

					<p>Fluctuations in division rate exhibit stochastic osillations (they are non-Markovian) </p>
					</section>

					<!-- ########################################################## -->
					<section>
					<h2>Results: beneficial mutations </h2>
					<p> Now let's consider the fate of a benificial mutation occuring in one cell. Can we use the argument from before but with the drift barrier at $N_{\rm eff} = 2\rho/3\ln(2)N$?   </p>
					<div class = "r-stack">
					<p class = "fragment current-visible"> Let $p_{\rm ext}$ be the probability that a mutant goes extinct. Then
					\[
					p_{\rm ext} = q +(1-q)p_{\rm ext} ^2.
				\]
					where
					\[q = {\mathbb P}[\text{cell is expelled before division}] =  \rho_r\int_0^{\infty}\int_0^{\tau_d}f_m(\tau_d)e^{-\rho_r \tau_e}d\tau_e d\tau_d = 1 - \tilde{f}_m(\rho_r)\]
					and $\tilde{f}_m(\rho_r) = \int_0^{\infty}f_m(\tau)e^{-\rho_r\tau}d\tau$ is the Laplace transform of $f_m$ evaluated at $\rho_r$.
				</p>
				<p class = "fragment current-visible"> If $q>1/2$, we get
					\[p_{\rm fix} = 1- p_{\rm ext} =1- \frac{q}{1-q} \]
					 which implies
					 \[\frac{1}{2-p_{\rm fix}} = \tilde{f}_m(\rho_r)\]
			</p>

					<img class = "fragment" src="./figures/el_vs_pfix.png" height = 300>
				</div>


					</section>


					<!-- ########################################################## -->
					<section>
					<h2>Results: beneficial mutations </h2>
					<p> <strong>Example</strong>: Let's assume the distribution of generation times is a gamma distribution. </p>
					<div class="container">
						<div class = "col">
						</br>
					<p class = "fragment"> We can show that
						\[
				p_{\rm fix} =  \frac{2}{{\rm CV}_{\tau}^2}\left(1-2^{-{\rm CV}_{\tau}^2} \right)s + O(s^2).\]</p>
				<p class= "fragment">Notice that as ${\rm CV}\to 0$, we obtain $p_{\rm fix} = 2\ln(2)s \ne N/N_{\rm eff}s$. </p>
			</div>
<div class = "col">
					<img class = "fragment" src="./figures/pfix_sims.png" height = 300></div></div>
					<div class = "fragment">


					<h4>We cannot capature the dynamics by rescaling Moran SDE -- connection between genetic drift and fixation breaks down. </h4></div>
				</section>

				<!-- ########################################################## -->
				<section>
				<h2>Results: long-term evolution </h2>
				<h4>Do small effects on fixation probability matter in the long-term? </h4>


				<p class = "fragment">  Recall that the time between mutations, and therefore the rate of evolution depends on the establishment time $\tau_{\rm est} = 1/(N\rho_r\mu p_{\rm fix})$</p>

				<p class = "fragment"> Imagine mutations increases growth rate via generation time ${\rm CV}$ but mutations have a fixed selective advantage $s$. In classical Moran model, fitness increases linearly.</p>
					<img class = "fragment" src="./figures/evo_traj.png" height = 300>

			</section>
		</section>

			<!-- ########################################################## -->

			<section>
				<h2>Future directions </h2>

			<p class = "fragment"> <strong>Correlated generation times</strong>: When generation times are correlated (the more biologically realistic scenerio) we get an integral equation for the extinction probability conditioned on the ancestral cells generation time:
					\[p_{\rm ext}(\tau) = q(\tau) + (1-q(\tau))\left[\int_0^{\infty}p_{\rm ext}(\tau')f(\tau'|\tau)d\tau'\right]^2 \] Haven't made much progress on this!  </p>

	<p class = "fragment"> <strong>Evolution in serial dilution experiments</strong>:
	In serial diluation experiments the situation is quite different because sampling occurs after a long period of growth. In this case a major source of fluctuations is the lag time-distribution, but many of the conceptual points we've made still apply (e.g. effects on long-term evolution).  </p>

	<p class = "fragment"> <strong>Predictions of genetic drift in continuous culture experiments</strong>:
   There has been some work measuring effective population size in continuous culture, but I think it would be cool to predict this based on knowledge of experimental parameters and single-cell traits. Can we predict how effective population size will respond to tuning e.g. dilution rate? That would be cool!   </p>
			</section>

				<!-- ########################################################## -->
			<section>
				<h2>Thanks for listening </h2>

					<img src="./figures/cover.svg">
					<h4>Questions? </h4>

					<p>Contact me for opportunities in my group!  </p>
			</section>


</section>




			</div>

		</div>

		<script src="./dist/reveal.js"></script>
		<script src="./plugin/math/math.js"></script>
		<script type="text/javascript" async
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-MML-AM_CHTML">
</script>
		<script>

			Reveal.initialize({
				center: false,
				history: true,
				transition: 'none',

				math: {
					// mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
					config: 'TeX-AMS_HTML-full',
					TeX: {
						Macros: {
							R: '\\mathbb{R}',
							set: [ '\\left\\{#1 \\; ; \\; #2\\right\\}', 2 ]
						}
					}
				},

				plugins: [ RevealMath ]
			});

		</script>



	</body>
</html>
