<!doctype html>
<html lang="en">

	<head>

		<meta charset="utf-8">

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<link rel="stylesheet" href="./dist/reveal.css">
		<link rel="stylesheet" href="./dist/theme/eal.css" id="theme">
	</head>
	<script src="https://d3js.org/d3.v4.min.js"></script>
	<script src="https://d3js.org/d3-timer.v1.min.js"></script>
	<script src="https://d3js.org/d3-random.v1.min.js"></script>
	<script src="https://d3js.org/d3-quadtree.v1.min.js"></script>
	<script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
	<style type="text/css">
    code {
      white-space: pre;
    }

    .MathJax_CHTML {
			color:rgb(100, 100, 100);
      font-size: 100% !important;
    }
		.container{
		display: flex;
		}
		.col{
		flex: 1;
		}


.vertical-center {
  margin: 0;
  position: absolute;
  top: 50%;
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
}


  </style>




	<body>

		<div class="reveal">

			<div class="slides">
				<!-- ########################################################## -->
				<section>
					<h2>Coarse graining evolutionary dynamics models</h2>
					 <!-- <img src="./figures/euler.png" height="400"> -->

					 	<img src="./figures/lineages1.svg">
					<h3>Ethan Levien</h3>

				</section>


<!-- ########################################################## -->
	<section>
		<h2> Selection acts on phsiological traits? </h2>

		<img src="./figures/scales.png">

		<p> The central challange in evolution is to connect processes across a range of spatial and temporal scales: evolution plays out at population level but is constrained by microscopic interactions</p>
		</section>

<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
		<section>
<!-- ########################################################## -->
        <section>
				<h2> Part 1: The population scale </h2>
					<div class="r-stack">
				<img src="./figures/turbidostat.png" height="400" data-fragment-index="1">
				<img class="fragment current-visible" src="./figures/eVOLVER.png" height="400" data-fragment-index="3">
			</div>
				<!-- <p>The turbidostat is a type of Continuous culture in which the population size is kept approximately constant through feedback between an optical density detector and a pump </p> -->
				<p class = "fragment" data-fragment-index="1">Let us begin by introducing evolutionary dynamics models without reference to the single-cell scale.  </p>
				<p class = "fragment" data-fragment-index="2">To keep the discussion biologically grounded, we will imagine evolution playing out in a continous culture.  </p>
			</section>



<!-- ########################################################## -->
			<section>
			<h2> Evolution in the turbidostat </h2>
			<p>
		The basic dynamics in a continous culture can be understood from the ODE model for the number of cells of each genotype $n_i$:
		\[
			\frac{dn_i}{dt} = \underbrace{\alpha_i(c)n_i}_{\text{growth}} - \underbrace{Dn_i}_{\text{dilution}}
			\]</p>
			<p class = "fragment" data-fragment-index="2">
				The growth rates depend on a limiting nutient:
				\[
			\frac{dc}{dt} = \underbrace{Dc_0}_{\text{media in}}- \underbrace{Dc}_{\text{dilution}}
- \underbrace{\gamma^{-1}\left[\sum_i\rho_i(c)n_i\right]c}_{\text{nurtient update}}
\] </p>

<div class="r-stack">
			<p class="fragment current-visible" data-fragment-index="3">In the <strong> turbidostat </strong> the dilution rate $D$ is adjusted to fix the total population size $N = \sum_i n_i$, which means
			\[
			\begin{equation}
			D = \rho_i(c)n_i \implies c  \to \frac{\gamma c_0}{\gamma+1}
			\end{equation}\]
		</p>

		<p class="fragment current-visible" data-fragment-index="4"> We obtain the so-called replicator equations for the genotype fractions $\phi_i = n_i/N$:
			\[
			\frac{d\phi_i}{dt} = \phi_i\left(\alpha_i - D - \sum_j \phi_j(\rho_j-D)\right)
		\]
	</p>

	<p class="fragment current-visible" data-fragment-index="5"> In general, dynamics are difficult to solve, but with two species these become logistic growth:
		\[
		\frac{d\phi}{dt} = (\rho_1 - \rho_2)\phi(1-\phi)
	\]
</p>
	</div>

				</section>


<!-- ########################################################## -->
				<section>
				<h2> The Moran process </h2>
				<p>These equations work well when $\phi$ is not too small, but new mutations emerge in single-cells, so $\phi(0) = 1/N$.</p>

				<h4 class = "fragment">Model must include fluctuations $\color{rgb(20, 116, 111)}{\implies}$ demands we make assumptions about single-cell dynamics </h4>
				<p class = "fragment"> <strong> Standard approach (Moran process): </strong> <p>
					<p> <ul> <li class = "fragment">Take cell division to be a Poission process; that is, time between cell divisions is exponentially distributed at a rate detemined by the genotypes growth rate: \[ p_i(\tau) = \rho_ie^{-\rho_i\tau}
				\]  </li>
				<li class = "fragment"> Every time a cell divides another cell is expelled from the culture in order to keep $N$ fixed. </li>
			</ul>
				</section>

				<!-- ########################################################## -->
				<section>
				<h2> The Moran process </h2>
					<img src="./figures/lineages2.svg" height="400">
				</section>

<!-- ########################################################## -->
				<section>
					<h2> The Moran process </h2>
					<p class = "fragment" data-fragment-index="1"> This means that over a small time interval $\Delta t$, the number of mutants gained and lost is
					\[\begin{aligned}\text{# of mutants gained} &\sim {\rm Poission}(N\rho_1\phi(1-\phi) \Delta t)\\
					\text{# of mutants lost} &\sim {\rm Poission}(N\rho_2\phi(1-\phi) \Delta t)
					\end{aligned}\]</p>

					<div class="r-stack">
					<p class="fragment current-visible" data-fragment-index="3"> We can calculate the mean and variance of $\Delta \phi$:
						\[\begin{aligned}\langle  \Delta \phi \rangle &= (\rho_1 - \rho_2)\phi(1-\phi)\Delta t\\
						{\rm var} \left(\Delta \phi \right) &= \frac{(\rho_1 + \rho_2)}{N}\phi(1-\phi)\Delta t
					\end{aligned}\]
				For simplicty we set $\rho_r = 1$ and $\rho_1 = 1+s$</p>
				<p class="fragment" data-fragment-index="4">
					Corresponds to SDE
					\[
					\frac{d}{dt}\phi = \underbrace{s\phi(1-\phi)}_{\text{selection}} + \underbrace{\sqrt{\frac{\phi(1-\phi)}{N}}\xi}_{\text{drift}}
					\]
				</p></div>

				<p class="fragment" data-fragment-index="4">
					or the Fokker-Plank equation
					\[
					\frac{\partial}{\partial}p(\phi,t) = -\frac{\partial}{\partial \phi}[s\phi(1-\phi)p(\phi,t)] + \frac{\partial^2}{\partial \phi^2}\left[\frac{\phi(1-\phi)}{N}p(\phi,t) \right]
					\]
				</p>

				</section>

				<!-- ########################################################## -->
				<section>
				<h2> Properties of the Moran model </h2>
						<p> Some trajectories starting on $\phi(0) = 1/N$  </p>
						<div class="r-stack">
						<img class="fragment" src="./figures/moran_fix1.svg" data-fragment-index="1">
						<img class="fragment" src="./figures/moran_fix2.svg" data-fragment-index="3" >
					</div>
						<p class="fragment" data-fragment-index="2" > The behavoir spends stronly on whether selection or drifts dominates. When does the transition occur? </p>

			</section>


			<!-- ########################################################## -->
			<section>
			<h2> Properties of the Moran model </h2>

			<div class="r-stack">
			<p class="fragment current-visible" data-fragment-index="1" > We know it must be when $\phi$ is small, so we make the approximation
				\[
		 \frac{d\phi}{dt} =s\phi(1-\phi) + \sqrt{\frac{\phi(1-\phi)}{N}} \approx s\phi + \sqrt{\phi/N}\xi
						\]
			 </p>

			 <p class="fragment" data-fragment-index="2" > Implies the PDE
				 \[
				\frac{\partial}{\partial}p(\phi,t) = -\frac{\partial}{\partial \phi}[s\phi p(\phi,t)] + \frac{\partial^2}{\partial \phi^2}\left[\frac{\phi}{N}p(\phi,t) \right]
				\]
			</p></div>


			<p class = "fragment" data-fragment-index = "3">
				We can use this to calculate the mean and variance
				\[
				\begin{aligned}
				\langle \phi \rangle &= \phi(0)e^{st}\\
				{\rm var}(\phi ) &= \phi(0)e^{st}(1-e)\\
					\end{aligned}
				\]
				 </p>

		 <p class = "fragment" data-fragment-index = "3">
			The coefficient of variation
			 \[
		    CV = \frac{{\rm std}(\phi)}{\langle \phi\rangle} =\frac{1-e^{-st}}{Ns\phi_0} \to \frac{1}{Ns \phi_0}
		  			\]
				</p>

				<p class = "fragment" data-fragment-index = "4">
				 $1/Ns$ is called the <strong> Drift barrier </strong>
					 </p>
			</section>


			<!-- ########################################################## -->
			<section>
			<h2> Properties of the Moran model </h2>

					<div class="r-stack">
					<img class="fragment" src="./figures/moran_fix1.svg" data-fragment-index="1">
					<img class="fragment" src="./figures/moran_fix2.svg" data-fragment-index="3" >
				</div>
					<div class="r-stack">
					<div class="fragment current-visible" data-fragment-index="2" > <p> <strong> Fluctuations dominate: </strong> This means $\langle \phi \rangle= 0$ and $\sqrt{\langle  \phi^2\rangle} \approx t/N\phi_0$. </p>
						<p> But if $\phi=0$ mutant is extinct forever, so really we expect $\phi = O(t/N\phi_0)$ or $\phi$ is extinct. Each unit time the mutant is equally likely to go exitinct or grow, so the chance of surviving until time $t$ is $1/t$. </p>
					</div>

					<div class="fragment current-visible" data-fragment-index="3" > <p> <strong> Selection dominates: </strong> This just logistic growth. </p>
						<p> <strong> Fxiation probability: </strong> The analysis tells us the chance to reach the drift barrier is $1/s$ </p>
					</div>
				</div>
		</section>



		<!-- ########################################################## -->
		<!-- <section>
		<h2> Properties of the Moran model </h2>
					<h4>So what is the chance the mutant lineage reaches the drift barrier before going extinct?</h4>

				<div class="r-stack">
		<p class="fragment current-visible">
		Below the drift barrier, we can approximate the dynamics over a small time $dt$ by
			\[
			\Delta \phi = \phi(t+\delta)-\phi(t) = \sqrt{\frac{\delta\phi(t)}{N}}Z,\quad Z \sim {\rm Normal}(0,1)
			\]
			This will approximation will give us an order-of-magnitude estimate of $\Delta \phi$ before some time $\delta^*$ when  $\delta^* = \phi(t)N$.
		</p>

		<p class="fragment current-visible">
		 At $\delta^*$, the mutant fraction will have changed by $\pm \phi(t)$.This means the population has roughly an equal chance of doubling or going extinct over a time period $\delta^*$. 	</p>

		 	<p class="fragment current-visible">
				We can repeat this approximation iteratively. After $k$ iterations, we find that  the population size is
			\[
			\phi(t_k) = \left\{ \begin{array}{cl}
			 2^k/N & \text{with prob. $1/2^k$}\\
			  0 & \text{with prob. $1-1/2^k$}\end{array}\right.
			\]
			where
			\[
			t_k = t_{k-1} + \phi(t_{k-1})N = \sum_j 2^{j} \sim 2^k \implies k = \ln(t)/\ln(2)
			\]
		</p>

		<p class="fragment current-visible">
	Plugging $k = \ln(t)/\ln(2)$ we are obtain order-of-magnitude estimates of the mutant size and chance of survival:
		\[
		\phi(t) = \left\{ \begin{array}{cl}
		 t/N & \text{with prob. $1/t$}\\
			0 & \text{with prob. $1-1/t$}\end{array}\right.
		\]
		So the population will reach a fraction $1/Ns = t/N$ after a time $t = 1/s$. The chance the survive until this time is $\sim s$.
	</p>

	</div>

	</section> -->
</section>

<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<section>
	<!-- ########################################################## -->
		<section>
			<h2> Selection acts on phsiological traits? </h2>

			<img src="./figures/scales.png">


			</section>

		<!-- ########################################################## -->
		<section>
		<h2>Motivation from single-cell data</h2>

						<iframe width="820" height="415"
		src="https://www.youtube.com//embed/5PyoO4S3BU0">
		</iframe>
		<p>  Wang et al. Current Biology (2010) </p>

		</section>

				<!-- ########################################################## -->
				<section>
				<h2>Motivation from single-cell data</h2>

				<p> A measure of an individual cells viability is its genertaion time. A cell with a shorter generation time will produce few offspring.  </p>
				<img class = "fragment" src="./figures/mm_drawing.png">
				<p class = "fragment"> Generation time is random, and correlated with many other traits. Natural to think about understanding adaptation in terms of $f(\tau)$ </p>
				</section>




				<!-- ########################################################## -->
				<section>
				<h2> Branching process model </h2>
				<p class = "fragment"> Start with simpler case: exponential growth with age structure  </p>
				<div class = "fragment">	<p> Alfred Lotka (1934), Bellman and Harris (1948) and Powell (1956): Each cell's generation time is drawn <b>independently</b> from a distribution $f(\tau)$.</p>	</div>

				<div class = "fragment">
				<div class="viewof-sigma" class="horizontal-center"></div>
				<div class="tree" ></div>
				<script type="module">
				import {Runtime, Inspector} from "https://cdn.jsdelivr.net/npm/@observablehq/runtime@4/dist/runtime.js";
				import define from "https://api.observablehq.com/@elevien/branching-process/2.js?v=3";
				(new Runtime).module(define, name => {
					if (name === "viewof sigma") return Inspector.into(".viewof-sigma")();
					if (name === "tree") return Inspector.into(".tree")();
					return ["tree_data1"].includes(name) || null;
				});
				</script></div>


				<div class = "fragment">
				<p> In the long-term, ${\mathbb E}[N(t)] = Ae^{\Lambda t}$ for some $\Lambda$. </p>
				</div>
				<div class = "fragment">
				<h4> How does the rate at which the population grows depend on the distribution of generation times? </h4>
				</div>



				</section>




				<!-- ########################################################## -->
				<section>
				<h2> Branching process model </h2>

				<div class="recursive_tree"></div>
				<script type="module">
				import {Runtime, Inspector} from "https://cdn.jsdelivr.net/npm/@observablehq/runtime@4/dist/runtime.js";
				import define from "https://api.observablehq.com/@elevien/branching-process/2.js?v=3";
				(new Runtime).module(define, name => {
				if (name === "recursive_tree") return Inspector.into(".recursive_tree")();
				});
				</script>


				<div class="fragment">
				<p>The root cell has two daughters with progenies that grow exponentially at the same rate $\Lambda$. </p></div>
				<div class="fragment"><p> Let $\tau$ be the first cell's generation time:
				<script type="math/tex; mode=display">
				{\mathbb E}\left[N_{\rm daughter}|\tau\right] = Ae^{\Lambda(t-\tau)} \implies
				{\mathbb E}\left[N_{\rm daughter}\right] = 	A \int_0^{\infty} e^{\Lambda(t-\tau)}f(\tau)d\tau
				</script>
				<!-- <script type="math/tex; mode=display">
				N_{\rm daughter}(t|\tau)= Ae^{\Lambda(t-\tau)} \implies
				{\mathbb E}_f\left[N_{\rm daughter}(t|\tau)\right] = 	A \int_0^{\infty} e^{\Lambda(t-\tau)}f(\tau)d\tau
				</script> -->
				</p>
				<div>

				<div class="fragment">
				<p> Writing the population size at time $t$ in terms of the size of both daughter progenies gives the so-called Euler-Lotka Formula
					<script type="math/tex; mode=display">
				Ae^{\Lambda t} = 	\color{rgb(20, 116, 111)}{{\mathbb E}\left[N_{\text{left daughter}}\right]} + \color{rgb(205, 88, 73)}	{{\mathbb E}\left[N_{\text{right daughter}}\right]}  = 2 A \int_0^{\infty} e^{\Lambda(t-\tau)}f(\tau)d\tau \implies \color{rgb(5, 5, 5)}{	\frac{1}{2} = \int_0^{\infty}f(\tau)e^{-\Lambda \tau}d\tau}
					</script>
				</p></div>




				</section>







					<!-- ########################################################## -->
					<section>
					<h2> Branching process in finite population </h2>
					<p> How can we connect dynamics at the single-cell level to population level dynamics?  </p>
				<p class="fragment">	\[
			 \frac{d}{dt} Q({\bf u},{\bf a},{\boldsymbol \lambda},{\bf s},t) + \sum_{k=1}^N \frac{\partial}{\partial a_k} Q({\bf u},{\bf a},{\boldsymbol \lambda},{\bf s},t)
			 = - Q({\bf u},{\bf a},{\boldsymbol \lambda},{\bf s},t)\left(D+ \sum_{k=1}^N \gamma_{u_k}(a_k,\lambda_k,s_{b,k})\right) \]
			along with the boundary conditions
			 \[
			Q(S_{k,u_v}({\bf u}),S_{k,0}(S_{v,0}({\bf a})),{\boldsymbol \lambda},
			S_{k,s_{b,v}e^{\lambda a_v}/2}(S_{v,s_{b,v}e^{\lambda a_v}/2}({\bf s})),t)\\
			\quad\quad= \frac{g(\lambda_k)g(\lambda_v)}{N}\int\int \gamma_{u_v}(a_v,\lambda_v,s_{b,v})
			\times Q({\bf u},{\bf a},S_{v,\lambda_v'}S_{k,\lambda_k'}(({\boldsymbol \lambda})),{\bf s},t)d\lambda_k'd\lambda_v'.\]</p>
			<p class="fragment">In these equations, $g(\lambda)$ is the distribution of single-cell growth rates and $\gamma$ is the division rate:
			\[
		\gamma_i(a,\lambda,s_b) = \frac{\int \int\, d\lambda\, ds_b\,  f_i(a|\lambda,s_b)}{1- \int_0^a\int \int\, d\lambda\, ds_b\,da'\,  f_i(a'|\lambda,s_b)}
	\] </p>

				</section>

						<!-- ########################################################## -->
						<section>
						<h2> Bridging-scales </h2>
						 <p> We don't really care about $Q$ though! we are interested in the behavoir of the mutant fraction $\phi$. </p>
						<h4 class="fragment"> Question: can we coarse-grain to get an effective SDE?  If so, what are the parameters? </h4>

						<p class="fragment"> Let's start by assuming a large mutant fraction with long-term growth rate $\Lambda_m$ competing with resident population with growth rate $\Lambda_r=1$. </p>

								<p class="fragment"> When $\phi$ is not "too small", we can approximate
						\[
						\frac{d}{dt}\phi = \frac{1}{N}\frac{d}{dt}m - \frac{\phi}{N}\frac{d}{dt}N = \Lambda_m \phi - \phi(\Lambda_m\phi +\Lambda_r (1-\phi) ) = s\phi(1-\phi)
					\] <p>

						<p class="fragment">
						We might guess that the corresponding SDE is something like
						\[
						\frac{d}{dt}\phi = s_{\rm eff}\phi(1-\phi) + \sqrt{\phi(1-\phi)/N_{\rm eff}}\xi
					\]
					This would imply $p_{\rm fix } \sim s\frac{N_{\rm eff}}{N}$
					 </p>



			</section>

							</section>



				<!-- ########################################################## -->
				<!-- ########################################################## -->
				<!-- ########################################################## -->
				<!-- ########################################################## -->
				<!-- ########################################################## -->
				<!-- ########################################################## -->
					<section>

					<!-- ########################################################## -->
					<section>
					<h2>Neutral dynamics </h2>
					<p> Recall that in the case where division times are expentially distributed, this is the classical Moran model </p>
					<p class = "fragment">
					\[
					{\rm var}(\phi) \approx \frac{t}{N}
				\] </p>

				<p> What about the case when generation times are perfectly deterministic? </p>

				<p> This would suggest that $N_{\rm eff}  = 4/3\ln(2)N$ and therefore $p_{\rm fix} \sim </p>
					</section>

					<!-- ########################################################## -->
					<section>
					<h2>The coalescent </h2>
					<p> We can understand the different with coelecent theory </p>
					</section>

					<!-- ########################################################## -->
					<section>
					<h2>Fixation probability </h2>
					<p> We can show
					\[
			p_{\rm fix} =  \frac{2}{{\rm CV}_{\tau}^2}\left(1-2^{-{\rm CV}_{\tau}^2} \right)s + O(s^2).\]</p>


					</section>

					<!-- ########################################################## -->
					<section>
					<h2>Fixation probability </h2>
					<p>Notice that as ${\rm CV}\to 0$, we obtain $p_{\rm fix} 2\ln(2)s \ne N/N_{\rm eff}s$. </p>

					<h4>We cannot capature the dynamics by rescaling Moran SDE -- connection between genetic drift and fixation breaks down. </h4>


					</section>
</section>

<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->
<!-- ########################################################## -->

<section>


						<!-- ########################################################## -->
						<section>
						<h2> The Euler-Lotka equation </h2>
						<p>	We have derived the <b>the Euler-Lotka equation.</b>
							<script type="math/tex; mode=display">
								\frac{1}{2} = \int_0^{\infty}f(\tau)e^{-\Lambda \tau}d\tau = {\mathbb E}_f\left[e^{-\Lambda \tau}  \right].
							</script>
						</p>

						<div class="fragment">

						<p> If all cells live for exactly a time $\tau$, then $N(t) = 2^{t/\tau}$, so the rate of exponential growth is
							<script type="math/tex; mode=display">
						\Lambda = \ln(2)/\tau
							</script></p></div>

						<div class="fragment">
						<p>
							<b>Jensen's inequality</b> tells us
							<script type="math/tex; mode=display">
							\frac{1}{2} = {\mathbb E}_f\left[e^{-\Lambda \tau}  \right] \ge e^{-\Lambda {\mathbb E}_f[\tau]}
							 \implies \Lambda \ge  \ln(2)/{\mathbb E}_f[\tau]
							</script>
						</p>
						</div>

						<div class="fragment">
						<h4> If we fix the average generation time then variability will increase the rate of growth. </h4>
						</div>


						<!-- <div class="fragment">
						<p>	Let's consider two limits:
						<ul>
							<li> All cells live for exactly a time $\tau_0$. Then we have
								<script type="math/tex; mode=display">
									\frac{1}{2} = e^{-\Lambda \tau_0} \implies \Lambda = \frac{\ln(2)}{\tau_0}
								</script>
							 </li>

							 <li> Cells have a constant probability $1/\tau_0$ of dividing per unit time. Then
								 <script type="math/tex; mode=display">
									 \frac{1}{2} = \int_0^{\infty}e^{-\tau/\tau_0-\Lambda \tau}d\tau \implies \Lambda =\frac{1}{\tau_0}
								 </script>
							 </li>
						</ul></p>
						</div> -->


						</section>


						<!-- ########################################################## -->
						<section>
						<h2>Motivation from single-cell data</h2>


						<img src="./figures/variability.png" >
						<p> Eldar and Elowitz, Nature (2010) </p>

						<!-- <div class="fragment">
						<h4> &ldquo;Do Differences Make a Difference?&rdquo; - Altschuler and Wu (2010)</h4>
					</div> -->
						</section>


						<!-- ########################################################## -->
						<!-- <section>
						<h2>Motivation from single-cell data</h2>

						<img src="./figures/persisters.png" >
						<p>  Balaban et al. Science (2004) </p>

						</section> -->










																	<!-- ########################################################## -->
																	<section>
																		<h2> Generalized Euler-Lotka Formula  </h2>

																		<p>	The Euler-Lotka formula has an elagent generalization to the case where cell's generation times are correlated with their ancestors (Lebowitz et al., 1974):
																			<script type="math/tex; mode=display">
																				\frac{1}{2} = \int_0^{\infty} \color{rgb(20, 116, 111)}{f_{\rm tree}(\tau)}e^{-\Lambda \tau}d\tau = {\mathbb E}_{\rm tree}\left[e^{-\Lambda \tau}  \right].
																			</script>
																			where $\color{rgb(20, 116, 111)}{f_{\rm tree}(\tau)}$ is the distribution of generation times <b> taken over all cells </b> throughout the history of an exponentially growing population.
																		</p>

																<div class="fragment">		<div class="tree_vs_lineage"></div>

						<script type="module">
						import {Runtime, Inspector} from "https://cdn.jsdelivr.net/npm/@observablehq/runtime@4/dist/runtime.js";
						import define from "https://api.observablehq.com/@elevien/lineage-representation.js?v=3";
						(new Runtime).module(define, name => {
						  if (name === "tree_vs_lineage") return Inspector.into(".tree_vs_lineage")();
						});
						</script>
						<p> In general $\color{rgb(20, 116, 111)}{f_{\rm tree}(\tau)} \ne  \color{rgb(205, 88, 73)}{f(\tau)}$ </p></div>
						<div class = "fragment">
						<p> $\implies$ does not connect growth rate to lineage dynamics!</p>
					</div>


									</section>
								</section>













			</div>

		</div>

		<script src="./dist/reveal.js"></script>
		<script src="./plugin/math/math.js"></script>
		<script type="text/javascript" async
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-MML-AM_CHTML">
</script>
		<script>

			Reveal.initialize({
				center: false,
				history: true,
				transition: 'none',

				math: {
					// mathjax: 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js',
					config: 'TeX-AMS_HTML-full',
					TeX: {
						Macros: {
							R: '\\mathbb{R}',
							set: [ '\\left\\{#1 \\; ; \\; #2\\right\\}', 2 ]
						}
					}
				},

				plugins: [ RevealMath ]
			});

		</script>



	</body>
</html>
